source/write/gridfs.txt

```diff
@@ -0,0 +1,265 @@
+.. _pymongo-gridfs:
+
+=================
+Store Large Files
+=================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+ 
+.. meta::
+   :keywords: binary large object, blob, storage
+
+Overview
+--------
+
+In this guide, you can learn how to store and retrieve large files in
+MongoDB by using **GridFS**. GridFS is a specification implemented by 
+{+driver-short+} that describes how to split files into chunks when storing them
+and reassemble them when retrieving them. The driver's implementation of
+GridFS is an abstraction that manages the operations and organization of
+the file storage. 
+
+You should use GridFS if the size of your files exceeds the BSON document
+size limit of 16MB. For more detailed information on whether GridFS is
+suitable for your use case, see :manual:`GridFS </core/gridfs>` in the
+MongoDB Server manual.
+
+See the following sections that describe GridFS operations and how to
+perform them:
```

Comment:
S: I'm not a huge fan of listing the upcoming sections of the page in the overview like this, especially when we have the "On this page" there. It seems like it adds unnecessary maintenance and reading for little value. I think a sentence like this would accomplish the same thing (without the list):

```suggestion
The following sections describe GridFS operations and how to
perform them.
```

Reply:
cool, i'm not either. just left these in from the previous page.